packages/celotool/src/cmds/deploy/destroy/oracle.ts

```diff
@@ -0,0 +1,16 @@
+import { createClusterIfNotExists, switchToClusterFromEnv } from 'src/lib/cluster'
+import { removeHelmRelease } from 'src/lib/oracle'
+import { DestroyArgv } from '../../deploy/destroy'
+
+export const command = 'oracle'
+
+export const describe = 'destroy the oracle package'
+
+export const builder = {}
+
+export const handler = async (argv: DestroyArgv) => {
+  await createClusterIfNotExists()
```

Comment:
You should be able to remove this line to decrease the run time-- if we're destroying we should assume that the cluster exists & if it doesn't there's no need in creating it